cargo-shuttle/src/client.rs

```diff
@@ -220,6 +223,12 @@ async fn to_api_result(res: Response) -> Result<DeploymentMeta> {
     let text = res.text().await?;
     match serde_json::from_str::<DeploymentMeta>(&text) {
         Ok(meta) => Ok(meta),
-        Err(_) => Err(anyhow!("{}", text)),
+        Err(error) => {
+            error!(
+                error = &error as &dyn std::error::Error,
+                text, "failed to parse deployment meta"
+            );
+            Err(anyhow!("could not parse server response"))
```

Comment:
Does using `error.context(anyhow!(...))` provide a nicer output?

Reply:
Yip, it gives a bit more of a "backtrace". Thanks!